frontend/app-preview/src/views/LandingPage.module.css

```diff
@@ -15,11 +19,17 @@
   border: none;
 }
 
-.iframeMobileViewContainer {
+.previewLimitationsInfo {
+  align-self: center;
+  width: 80%;
+  margin: 10px;
+}
+
+.iframeContainer {
   display: flex;
   justify-content: center;
   background-color: #efefef;
-  height: calc(100vh - var(--header-height));
+  height: calc(100vh - var(--header-height) - var(--previewLimitationsInfo-height));
```

Comment:
Er formålet med previewLimitationsInfo-height bare å beregne riktig størrelse for iframen? 😊

Jeg legger merke til at previewLimitationsInfo får sine verdier ved å hente høyden fra alerten via document-apiet.

Jeg vil gjerne diskutere to ideer knyttet til denne tilnærmingen. Den første ideen er at det kanskje er lurt å unngå å bruke document-apiet direkte når vi jobber med React. Det kan være gunstig å benytte seg av Reacts toppnivå-api, som i dette tilfellet innebærer å bruke en referanse (ref) fra Alert-komponenten.

En annen tanke jeg har er om det er mulig å la iframens høyde skalere automatisk uten behov for manuelle beregninger? Det er kanskje ikke så lett å la iframe-skallere basert på parent-container? 🤔